File: Selenium/Project/Python/Activity4.py
import pytest
from selenium.webdriver.common.by import By

@pytest.mark.usefixtures("setup")
class TestSite:

    def test_verify_second_heading(self):
        self.driver.get("https://alchemy.hguy.co/jobs/")
        secondHeading = self.driver.find_element(By.XPATH, "//h2").text
        print(secondHeading)
        assert secondHeading == "Quia quis non"

File: Selenium/Project/Java/Activity4.java
package project;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity4 {
    WebDriver driver;
    @BeforeClass
    public void setUp()
    {
        WebDriverManager.firefoxdriver().setup();
        driver = new FirefoxDriver();
        driver.get("https://alchemy.hguy.co/jobs/");
    }
    @Test
    public void verifySecondHeading()
    {
        String secondHeading = driver.findElement(By.xpath("//h2")).getText();
        Assert.assertEquals(secondHeading,"Quia quis non");
    }
    @AfterClass
    public void closeBrowser()
    {
        driver.close();
    }
}
